apps/taxes/src/modules/avatax/avatax-client.ts

```diff
@@ -112,4 +118,16 @@ export class AvataxClient {
       filter: `code eq ${useCode}`,
     });
   }
+
+  async fullyRefundTransaction(params: RefundTransactionParams) {
+    return this.client.refundTransaction({
+      transactionCode: params.transactionCode,
+      companyCode: params.companyCode,
+      model: {
+        refundTransactionCode: params.transactionCode,
+        refundDate: new Date(),
```

Comment:
shouldn't date be taken from Saleor event? What if Saleor queue is full and webhook is delivered in 3h?

Reply:
+1

```graphql
subscription {
  event {
    ... on OrderFullyRefunded {
      issuedAt
    }
  }
}
```